testfiles/CMakeLists.txt

```diff
@@ -688,23 +688,23 @@ add_simulation_test(IDF_FILE WCE_Interior_VB_-45_deg.idf EPW_FILE USA_IL_Chicago
 add_simulation_test(IDF_FILE _5ZoneAirCooled_annual.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw ANNUAL_SIMULATION)
 add_simulation_test(IDF_FILE _5ZoneAirCooled_LeapYear_annual.idf EPW_FILE MadeUpLeapYear.epw ANNUAL_SIMULATION)
 
-# ASHRAE 90.1-2016 "Prototype Buildings"
-add_simulation_test(IDF_FILE ASHRAE9012016_ApartmentHighRise_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_ApartmentMidRise_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_Hospital_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_HotelLarge_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_HotelSmall_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_OfficeLarge_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_OfficeMedium_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_OfficeSmall_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_OutPatientHealthCare_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_RestaurantFastFood_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_RestaurantSitDown_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_RetailStandalone_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_RetailStripmall_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_SchoolPrimary_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_SchoolSecondary_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_Warehouse_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
+# ASHRAE 90.1-2019 DOE commercial prototype models
```

Comment:
So we are deleting the 90.1 2016 prototypes from this repo. That's OK, but are these files saved somewhere for future use? A DOE web site?

Reply:
All versions of the 90.1 prototype models are saved at the DOE's Building Energy Codes Program [website](https://www.energycodes.gov/prototype-building-models) in the zip files.